### ISSUE TITLE
Form input element event handlers do not bind properly in IE 11

### ISSUE THREAD

**Issue by jasonmevans** [emberjs/ember.js#16674]
This is probably an IE issue, but figure best to log it here.

I've also put together a twiddle to illustrate the point, but it's difficult to validate the issue since it looks like twiddle doesn't work in IE 11/Edge due to lack of the browser's support for `srcdoc`. However, I put together the same demo in a fresh ember test app on `localhost` where I could validate the issue in IE 11 (also validated in Edge where it works normally).

https://gist.github.com/jasonmevans/f845611292bbd7a486c7090dd2e29dfa

https://ember-twiddle.com/f845611292bbd7a486c7090dd2e29dfa

In a nutshell, binding an action to either a `{{input}}` helper or a `<input>` element does not work in IE 11. 

Here is a screen shot of the console errors:
<img width="430" alt="screen shot 2018-05-22 at 12 54 10 pm" src="https://user-images.githubusercontent.com/505818/40394225-6b130d0a-5dd7-11e8-9d05-b876e70f3322.png">

**Comment by jasonmevans** [comment]
Discovered root cause appears to be in how happypack is being used in our build. Looks like the es2015 Babel plugin isn't being initialized properly. Closing this issue.
